harness/tests/test/unit/test_coretypes_basis.gd

```diff
@@ -22,3 +22,34 @@ func test_should_set_right_vector3_using_operator_set():
 	jvm_basis = basis_test.set(jvm_basis, 2, Vector3(50, 90, 70))
 	assert_eq(engine_basis, jvm_basis, "Basis set operator on JVM side should be same as gdscript one.")
 	basis_test.free()
+
+func test_jvm_basis_get_rotation_quat():
+	var basis_test = godot_tests_coretypes_BasisTest.new()
+	var basis = Basis.IDENTITY.rotated(AXIS_X, 90.0 * PI / 180.0).scaled(Vector3(2.0, 2.0, 2.0))
+	var jvm_quat = basis_test.get_rotation_quat(basis) # should not throw, was throwing before
+	var engine_quat = Quat(AXIS_X, 90.0 * PI / 180.0)
+	assert_true(engine_quat.is_equal_approx(jvm_quat), "Quat from Basis in JVM should equal expected Quat")
+	basis_test.free()
+
+func test_new_jvm_basis_should_be_unity():
+	var basis_test = godot_tests_coretypes_BasisTest.new()
+	var jvm_basis = basis_test.new_jvm_basis()
+	var engine_basis = Basis(Vector3(1, 0, 0), Vector3(0, 1, 0), Vector3(0, 0, 1))
+	assert_eq(engine_basis, jvm_basis, "Basis() on JVM side should be same as unity gdscript one.")
+	basis_test.free()
+
+func test_jvm_basis_is_equal_approx():
```

Comment:
NIT: I'd add the issue link as a comment to these tests, so we know in the future why these were added before without needing to go through the commit history.

Reply:
I added links referencing this PR